docs/source/plugin/anatomy.rst

```diff
@@ -80,6 +80,18 @@ better to limit who can trigger them. There are decorators for that:
   trigger the rule
 * :func:`sopel.plugin.require_owner`: only the bot's owner can trigger the rule
 
+Sometimes it's not the privileges of the user who triggers a command that
+matters, but the **bot's privilege level** in a channel. For that, there are
+two options:
```

Comment:
```suggestion
Sometimes it's not the channel privilege level of the user who triggers a
command that matters, but the **bot's** privilege level. For that, there are
two options:
```

I'm just spitballing here, mostly to avoid the confusing "does-it-need-an-S-or-not" issue with the old phrasing. 😁